Gems/ROS2/Code/Include/ROS2/ROS2Bus.h

```diff
@@ -9,6 +9,7 @@
 
 #include <AzCore/EBus/EBus.h>
 #include <AzCore/Interface/Interface.h>
+#include <AzCore/std/string/string.h>
```

Comment:
This file doesn't seem to reference `AZStd::string`, is it actually needed?

Reply:
No, it's not. Thanks!